packages/neuron-ui/src/tests/formatters/shannonToCKBFormatter/index.test.ts

```diff
@@ -10,6 +10,6 @@ describe(`Verify shannon to CKB formatter`, () => {
   })
 
   test.each(fixtureTable)(`%s shannons => %s CKB with sign`, (shannons: string, expected: string) => {
-    expect(shannonToCKBFormatter(shannons, true)).toBe(+shannons > 0 ? `+${expected}` : expected)
+    expect(shannonToCKBFormatter(shannons, true)).toBe(+shannons >= 0 ? `+${expected}` : expected)
```

Comment:
Refactor should not update the function

Reply:
OK. Let me close this one and create a new one.